js/image/shared/Image.svelte

```diff
@@ -1,5 +1,6 @@
 <script lang="ts">
 	import type { HTMLImgAttributes } from "svelte/elements";
+	import DOMPurify from "dompurify";
```

Comment:
We have our own special sanitizer package that works in both server and client environments. See import here: https://github.com/gradio-app/gradio/blob/main/js/markdown-code/MarkdownCode.svelte#L6.

We need to 

```js
import { sanitize } from "@gradio/sanitize";
```

You can see the usage here:https://github.com/gradio-app/gradio/blob/main/js/markdown-code/MarkdownCode.svelte#L62

Reply:
Thanks @pngwn I'll switch to that. We use dompurify directly here:

https://github.com/gradio-app/gradio/blob/f3bedd4011bdfdecc952eb1275a9dd96af3e8d71/js/statustracker/static/ToastContent.svelte#L3

Does this need to be switched out too?

Reply:
Yeah, we should for consistency, although its less problematic in that case because we only launch toasts on the client.